lib/jxl/dec_xyb-inl.h

```diff
@@ -86,10 +86,34 @@ static inline HWY_MAYBE_UNUSED bool HasFastXYBTosRGB8() {
 #endif
 }
 
-static inline HWY_MAYBE_UNUSED void FastXYBTosRGB8(
-    const Image3F& input, const Rect& input_rect, const Rect& output_buf_rect,
-    const ImageF* alpha, const Rect& alpha_rect, bool is_rgba,
-    uint8_t* JXL_RESTRICT output_buf, size_t xsize, size_t output_stride) {
+template <size_t shift>
```

Comment:
Would be good to explain the requirements of this function; such as `in` needs xsize rounded up to 4 floats; and `out`needs round up to 8 int16_t, right?

Reply:
No need for that for out - no more than xsize outputs are written.